marshal/base-types.c

```diff
@@ -128,9 +134,25 @@ type##_Unmarshal ( \
          (uintptr_t)buffer, \
          (uintptr_t)dest, \
          local_offset); \
-    type tmp = 0; \
+\
     memcpy (&tmp, &buffer [local_offset], sizeof (tmp)); \
```

Comment:
Above you removed the space between `sizeof` and `(` but not here. If we're going to change the style we should change them all (at least in this module).